src/Apps/Artist/ArtistApp.tsx

```diff
@@ -1,184 +1,71 @@
 import { Jump } from "Utils/Hooks/useJump"
-import { useSystemContext } from "System/SystemContext"
-
-/**
- * For logged-out users, the sign-up modal is triggered via a global listener.
- * @see https://github.com/artsy/force/blob/136e7087233b5e482fa69fd00d4f1050e492c17c/src/client.tsx#L22-L25
- * @see https://github.com/artsy/force/blob/136e7087233b5e482fa69fd00d4f1050e492c17c/src/middleware.ts#L150
- */
 
 interface ArtistAppProps {
   artist: ArtistApp_artist$data
-  match: Match
 }
 
-const ArtistApp: React.FC<ArtistAppProps> = ({ artist, children, match }) => {
-  const route = findCurrentRoute(match)!
-  const artworkId = match.params.artworkId
-  const { isEigen } = useSystemContext()
+const ArtistApp: React.FC<ArtistAppProps> = ({ artist, children }) => {
+  const { contextPageOwnerType, contextPageOwnerSlug } = useAnalyticsContext()
 
   useScrollToOpenArtistAuthModal()
 
-  // FIXME: Instead of polluting the global route config, utilize two different parent apps
-  // Sub-page with a back button
-  if (route.hideNavigationTabs) {
-    return (
-      <>
-        {!isEigen && (
-          <BackLinkFragmentContainer artist={artist} artworkId={artworkId} />
-        )}
-
-        <PageWrapper artist={artist}>{children}</PageWrapper>
-      </>
-    )
-  }
+  return (
+    <>
+      <ArtistMetaFragmentContainer artist={artist} />
 
-  const showOverviewTab = hasOverviewContent(artist)
-  const showArtworksTab = artist.statuses?.artworks
-  const showAuctionResultsTab = artist.statuses?.auctionLots
+      <AnalyticsContext.Provider
+        value={{
+          contextPageOwnerId: artist.internalID,
+          contextPageOwnerSlug,
+          contextPageOwnerType,
+        }}
+      >
+        <Spacer y={[2, 4]} />
 
-  // Default page
-  return (
-    <PageWrapper artist={artist}>
-      <ArtistHeaderFragmentContainer artist={artist} />
+        <ArtistHeaderFragmentContainer artist={artist} />
 
-      <Spacer y={[4, 12]} />
+        <Spacer y={[4, 12]} />
 
-      <Jump id="artistContentArea" />
+        <Jump id="artistContentArea" />
 
-      <RouteTabs mb={2} fill data-test="navigationTabs">
-        {showOverviewTab && (
+        <RouteTabs fill data-test="navigationTabs">
           <RouteTab exact to={`/artist/${artist.slug}`}>
-            Overview
+            Artworks
           </RouteTab>
-        )}
 
-        {showArtworksTab && (
-          <RouteTab to={`/artist/${artist.slug}/works-for-sale`}>
-            {artist?.counts?.forSaleArtworks! > 0
-              ? `Works for Sale (${artist?.counts?.forSaleArtworks?.toLocaleString()})`
-              : "Artworks"}
-          </RouteTab>
-        )}
-
-        {showAuctionResultsTab && (
           <RouteTab to={`/artist/${artist.slug}/auction-results`}>
             Auction Results
           </RouteTab>
-        )}
-      </RouteTabs>
 
-      <Box pt={4}>{children}</Box>
-    </PageWrapper>
-  )
-}
+          <RouteTab to={`/artist/${artist.slug}/about`}>About</RouteTab>
```

Comment:
also here, just a general questions about hiding routes that are WIP

Reply:
after taking a second look I see that there's a review app being built for the tabs so I'm assuming we wouldn't need to rename routes?